pkg/metricsservice/client.go

```diff
@@ -37,17 +37,23 @@ type GrpcClient struct {
 }
 
 func NewGrpcClient(url, certDir, authority string) (*GrpcClient, error) {
-	defaultConfig := `{
-		"methodConfig": [{
-		  "timeout": "3s",
-		  "waitForReady": true,
-		  "retryPolicy": {
-			  "InitialBackoff": ".25s",
-			  "MaxBackoff": "2.0s",
-			  "BackoffMultiplier": 2,
-			  "RetryableStatusCodes": [ "UNAVAILABLE" ]
-		  }
-		}]}`
+	defaultConfig := fmt.Sprintf(`{
+			"methodConfig": [{
+			"timeout": "3s",
+			"waitForReady": true,
+			"retryPolicy": {
+				"InitialBackoff": ".25s",
+				"MaxBackoff": "2.0s",
+				"BackoffMultiplier": 2,
+				"RetryableStatusCodes": [ "UNAVAILABLE" ]
+			}
+			}],
+			"loadBalancingPolicy": "round_robin",
```

Comment:
@JorTurFer This is how the client will choose the right instance. It will round-robin between servers until it gets a server returning a SERVING result for the health check on the metrics-service